epub33/core/index.html

```diff
@@ -1327,9 +1327,9 @@ <h4>Intrinsic fallbacks</h4>
 						<h5>HTML <code>audio</code> and <code>video</code> fallbacks</h5>
 
 						<p id="confreq-resources-cd-fallback-media">[=EPUB creators=] MUST NOT use embedded
-							[[html]]&#160;[=flow content=] within a <a data-cite="html#the-media-element">media
+							[[html]]&#160;[=flow content=] within a <a data-cite="html#media-elements">media
 								element</a> (i.e, <a data-cite="html#the-audio-element"><code>audio</code></a> or <a
-								data-cite="html#the-vido-element"><code>video</code></a>) as an intrinsic fallback for
+								data-cite="html#the-video-element"><code>video</code></a>) as an intrinsic fallback for
```

Comment:
FWIW you could use the respec shortlink syntax `[^video^]` (and likewise `[^audio^]`)

Reply:
For the `video` element, yes, but not for `audio` (the markup for `audio` probably got copied when this section was tweaked not too long ago).

Because media overlays defines an `audio` element, the shorthand syntax always links to it. It doesn't matter if you put a `data-cite` attribute in, either. We reported the bug here: https://github.com/w3c/respec/issues/4133#issuecomment-1128788448